tools/python_integration_tests/slurm_simple_job_completion.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2024 "Google LLC"
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+from classes.ssh import SSHManager
+from classes.deployment import Deployment
+from classes.test import SlurmSimpleJobCompletionTest
+
+def test_topology(blueprint: str):    
+    deployment = Deployment(blueprint) 
+    try:
+        test = SlurmSimpleJobCompletionTest(deployment)
+        test.setup_deployment()
+        test.ssh_login()
+        test.run_test()
+    finally:
+        test.deployment.destroy()
+        test.close_ssh()
+
+def main(blueprint_yaml) -> None:
+    test_topology(blueprint_yaml)
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("blueprint_yaml", help="File path to blueprint to run topology test on.")
```

Comment:
Why do we need to supply blueprint, AFAIK those tests are tailored to specific blueprint. Lets hardcode it.

Reply:
Hardcoded with the blueprints I have been using to test with (added them to the blueprints folder). Can probably update to existing examples later.